ui/src/table.tsx

```diff
@@ -444,21 +453,26 @@ const
         }
       },
       onRenderRow = (props?: Fluent.IDetailsRowProps) => props
-        ? <Fluent.DetailsRow {...props} styles={{
+        ? <Fluent.DetailsRow {...props} styles={styleProps =>
+        ({
           cell: { alignSelf: 'center', fontSize: 14, lineHeight: 20, color: cssVar('$text9') },
           checkCell: { display: 'flex', alignItems: 'center' },
-          root: {
-            width: '100%',
-            border: border(2, 'transparent'),
-            borderTop: border(2, cssVar('$neutralLight')),
-            background: cssVar('$card'),
-            minHeight: 48,
-            '&:hover': {
-              background: cssVar('$neutralLight'),
-              border: `${border(2, cssVar('$themePrimary'))} !important`,
-            }
-          }
-        }} />
+          root: [
+            {
+              width: '100%',
+              border: border(2, 'transparent'),
+              borderTop: border(2, cssVar('$neutralLight')),
+              background: cssVar('$card'),
+              minHeight: 48,
+              '&:hover': {
+                background: cssVar('$neutralLight'),
+                border: `${border(2, cssVar('$themePrimary'))} !important`
+              }
+            },
+            styleProps.isSelected ? { background: cssVar('$neutralLight') } : undefined
```

Comment:
Can we use a different color? Since this one is already used for the table header/footer.

<img width="1646" alt="image" src="https://user-images.githubusercontent.com/64769322/204798958-c946f54b-bdbe-4b42-be7a-76c4610be784.png">